src/test/java/teammates/test/cases/automated/SubmissionsAdjustmentTest.java

```diff
@@ -1,337 +0,0 @@
-import teammates.common.datatransfer.StudentEnrollDetails;
-import teammates.common.datatransfer.StudentUpdateStatus;
-import teammates.common.exception.EnrollException;
-import teammates.common.exception.EntityDoesNotExistException;
-import teammates.common.util.Const;
-import teammates.common.util.Const.ParamsNames;
-import teammates.common.util.FieldValidator;
-import teammates.common.util.HttpRequestHelper;
-import teammates.common.util.JsonUtils;
-import teammates.common.util.Sanitizer;
-import teammates.logic.core.AccountsLogic;
-import teammates.logic.core.CoursesLogic;
-import teammates.logic.core.FeedbackQuestionsLogic;
-import teammates.logic.core.FeedbackResponsesLogic;
-import teammates.logic.core.FeedbackSessionsLogic;
-import teammates.logic.core.StudentsLogic;
-
-import com.google.appengine.api.urlfetch.URLFetchServicePb.URLFetchRequest;
-
-public class SubmissionsAdjustmentTest extends BaseComponentUsingTaskQueueTestCase {
-    
-    protected static StudentsLogic studentsLogic = StudentsLogic.inst();
-    protected static FeedbackResponsesLogic frLogic = FeedbackResponsesLogic.inst();
-    protected static FeedbackSessionsLogic fsLogic = FeedbackSessionsLogic.inst();
-    protected static AccountsLogic accountsLogic = AccountsLogic.inst();
-    protected static CoursesLogic coursesLogic = CoursesLogic.inst();
-    private static DataBundle dataBundle = getTypicalDataBundle();
-    
-    @SuppressWarnings("serial")
-    public static class SubmissionsAdjustmentTaskQueueCallback extends BaseTaskQueueCallback {
-        
-        @Override
-        public int execute(URLFetchRequest request) {
-            HashMap<String, String> paramMap = HttpRequestHelper.getParamMap(request);
-            
-            assertTrue(paramMap.containsKey(ParamsNames.COURSE_ID));
-            assertNotNull(paramMap.get(ParamsNames.COURSE_ID));
-            
-            assertTrue(paramMap.containsKey(ParamsNames.ENROLLMENT_DETAILS));
-            assertNotNull(paramMap.get(ParamsNames.ENROLLMENT_DETAILS));
-            
-            assertTrue(paramMap.containsKey(ParamsNames.FEEDBACK_SESSION_NAME));
-            assertNotNull(paramMap.get(ParamsNames.FEEDBACK_SESSION_NAME));
-            
-            SubmissionsAdjustmentTaskQueueCallback.taskCount++;
-            return TASK_QUEUE_RESPONSE_OK;
-        }
-
-    }
-    
-    @BeforeClass
-    public static void classSetUp() throws Exception {
-        printTestClassHeader();
-        gaeSimulation.tearDown();
-        gaeSimulation.setupWithTaskQueueCallbackClass(
-                SubmissionsAdjustmentTaskQueueCallback.class);
-        gaeSimulation.resetDatastore();
-        removeAndRestoreTypicalDataInDatastore();
-    }
-    
-    @AfterClass
-    public static void classTearDown() {
-        printTestClassFooter();
-    }
-    
-    @Test(enabled = false)
-    public void testEnrollStudentsWithScheduledSubmissionAdjustment() throws Exception {
```

Comment:
`InstructorCourseEnrollSaveActionTest.java` because that action is the one that triggers the task in question. It becomes as simple as `verifySpecificTasksAdded` or `verifyNoTasksAdded`. I'm not sure if the coverage level is the same, but this is another issue.